### DISCUSSION TITLE
default_bind_group ghost?

### DISCUSSION THREAD

**Discussion by GeoffNT** [Koenkk/zigbee2mqtt#13117] [General]
I have been testing an IKEA on/off remote, but have since deleted it.
In my groups list though I have:
Group ID 901 default_bind_group with 0 group members and 0 group scenes. If I try to delete it I get the error message Failed with error.  "Group default_bind_group does not exist.

I have tried restarting zigbee2mqtt but still get the same error. 

It doesn't seem to be causing any issues but I don't like it being there/not there at the same time! 

Is there a way to force delete this somehow?

**Comment by MichaelUray** [comment]
Did you actually figure that out?

**Comment by MSL-DA** [comment]
I see the same problem and cannot delete the group in version 2.0

<img width="1719" alt="Skærmbillede 2024-12-23 kl  22 02 54" src="https://github.com/user-attachments/assets/9f8337de-c6f1-4ff0-89d9-23276eace252" />

<img width="1719" alt="Skærmbillede 2024-12-23 kl  22 03 11" src="https://github.com/user-attachments/assets/f5081d4c-16ce-4665-8193-b5a7a3050b34" />

**Comment by MSL-DA** [comment]
Well ... I have deleted the group in database.db
